lib/matplotlib/ticker.py

```diff
@@ -418,16 +486,20 @@ def pprint_val(self, x, d):
 
 class ScalarFormatter(Formatter):
     """
-    Tick location is a plain old number.  If useOffset==True and the data range
-    is much smaller than the data average, then an offset will be determined
-    such that the tick labels are meaningful. Scientific notation is used for
-    data < 10^-n or data >= 10^m, where n and m are the power limits set using
-    set_powerlimits((n,m)). The defaults for these are controlled by the
-    axes.formatter.limits rc parameter.
-
+    Format tick values as a number.
     """
-
     def __init__(self, useOffset=None, useMathText=None, useLocale=None):
+        """
```

Comment:
Does it make sense to document both class and `__init__`? I think Sphinx has a setting for it, so it may ignore one or the other, or even just combine both together.